km52/Mykhailenko_Yeva/laboratory1.sql

```diff
@@ -1,2 +1,120 @@
 -- LABORATORY WORK 1
--- BY Mykhailenko_Yeva
+-- BY Mykhaylenko_Yeva
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+вставляти дані до таблиць
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+CREATE USER mykhaylenko IDENTIFIED BY mykhaylenko
+QUOTA 100M ON USERS; 
+
+GRANT ALTER ANY TABLE TO mykhaylenko;
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Класна кімната має парти та стільці.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
```

Comment:
# **Entity table_**:
id_table (unique desk identification number, primary key)
REGEXP : ^[0-9]{6}$
EXAMPLE: 011111, 123456

table_color
REGEXP :^#[a-f0-9]{6}$
EXAMPLE: #F3F3F3, #BC442A, #08113D

size_table
REGEXP : ^[0-9]{3}$
EXAMPLE: 123, 001

date_table
REGEXP : ^[0-9]{4}[-]{1}[0-9]{2}[-]{1}[0-9]{2}$
EXAMPLE: 2012-12-12, 1998-02-11

# Entity chair:
id_chair (unique chair identification number, primary key)
REGEXP : ^[0-9]{6}$
EXAMPLE: 011111, 123456

chair_color
REGEXP :^#[a-f0-9]{6}$
EXAMPLE: #F3F3F3, #BC442A, #08113D

size_chair
REGEXP : ^[0-9]{3}$
EXAMPLE: 111, 123

date_chair
REGEXP : ^[0-9]{4}[-]{1}[0-9]{2}[-]{1}[0-9]{2}$
EXAMPLE: 2012-12-12, 1998-02-11


# Entity room:
table_id(corresponds to field id_table in table_)
REGEXP : ^[0-9]{6}$
EXAMPLE: 011111, 123456

chair_id(corresponds to field id_chair in chair)
REGEXP : ^[0-9]{6}$
EXAMPLE: 011111, 123456

room_number
REGEXP : ^[0-9]{3}$
EXAMPLE: 001, 125

room_area 
REGEXP : ^[0-9]{3}$
EXAMPLE: 001, 999